src/components/Course.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import SidebarMenu from './SidebarMenu'
+import { Link } from '@reach/router'
+import Lesson from './Lesson'
+import IntroLesson from './IntroLesson'
+
+const Course = ({ course, lessonSlug }) => {
+  const lessons = course.lessonsCollection.items
+  let matchingLesson = null
+  let nextLesson = null
+  // prep sidebar
+  const sidebarLinks = [
+    <div key='course-overview' className='table-of-contents__item'>
+      <Link className='table-of-contents__link' to={`/courses/${course.slug}` /* +query??? */}>Course overview</Link>
+    </div>
+  ]
+
+  lessons.forEach(lesson => {
```

Comment:
Yeah i think it's better. I did `forEach` because I hardcoded the first element in the list to be the course overview. With `map` I'll just unshift after I do the map to add it to the front of the list. I still think it's an improvement